### PR TITLE
Added CreateOnUpdate and applied formating

### PR DESCRIPTION
**Pull request by bab29**
Allow for creating accounts when they don't exist when running on update mode.
Checks to see if current secret matches the secret being supplied

Implemented Changes
Updated to allow for creation of accounts when you run in update.
Checks to see if current secret matches the secret to be changed to to prevent excessive versions and reduce load on CPM

Connected Issue/Story
None

Definition of Done
At least 1 todo must be completed in the sections below for the PR to be
merged.

Changelog
[x ] The CHANGELOG has been updated, or
This PR does not include user-facing changes and doesn't require a
CHANGELOG update
Test coverage
This PR includes new unit and integration tests to go with the code
changes, or
[ x] The changes in this PR do not require tests
Documentation
[ x] Docs (e.g. READMEs) were updated in this PR
A follow-up issue to update official docs has been filed here: insert issue ID
This PR does not require updating any documentation
Behavior
[ x] This PR changes product behavior and has been reviewed by a PO, or
These changes are part of a larger initiative that will be reviewed later, or
No behavior was changed with this PR
Security
Security architect has reviewed the changes in this PR,
These changes are part of a larger initiative with a separate securit

### CODE REVIEW

File: Account Onboard Utility/Accounts_Onboard_Utility.ps1 [hunk 1]
```diff
@@ -1292,388 +1198,320 @@ Write-LogMessage -Type Info -MSG "Getting PVWA Credentials to start Onboarding A
...
 	}
+}
 #endregion
 
+#region Check Create on Update
+
+if ($CreateOnUpdate) {$Create = $CreateOnUpdate}
+
+#endRetion
+
```
Comment (bab29) [inline Account Onboard Utility/Accounts_Onboard_Utility.ps1:1250]: Added code

File: Account Onboard Utility/Accounts_Onboard_Utility.ps1 [hunk 2]
```diff
@@ -65,7 +69,10 @@ param(
 	
 	# Use this switch to disable Auto-Update
 	[Parameter(Mandatory=$false)]
-	[Switch]$DisableAutoUpdate
+	[Switch]$DisableAutoUpdate,
+
+	[Parameter(ParameterSetName='Update',Mandatory=$false)]
+	[Switch]$CreateOnUpdate
```
Comment (bab29) [inline Account Onboard Utility/Accounts_Onboard_Utility.ps1:75]: Added code

**Review by bab29 — COMMENTED** [review]
Marked added code

**Comment by AssafMiron** [comment]
Looks good :)
Thanks

**Pull request merged**